lib/widgets/subscription_list.dart

```diff
@@ -276,6 +282,15 @@ class SubscriptionItem extends StatelessWidget {
                 count: unreadCount,
                 backgroundColor: swatch,
                 bold: true)),
+          ] else if (hasOnlyMutedMentions && !subscription.isMuted) ...[
+            const SizedBox(width: 12),
+            const AtMentionMarker(muted: true),
+            Opacity(
+              opacity: mutedOpacity,
+              child: UnreadCountBadge(
+                count: mutedUnreadCount,
+                backgroundColor: swatch,
+                bold: true)),
```

Comment:
Looks like the later part is mostly identical to the previous branch except for the opacity/unread count. I think it would be beneficial to find a way that do not couple `AtMentionMarker` with `UnreadCountBadge`, so that each has individual logic, making the change easier to verify.

Reply:
For example, we can have something like (not sure if the conditions are right):
```dart
          if (hasMentions) ...[
            AtMentionMarker(muted: hasOnlyMutedMentions || subscription.isMuted)
          ]
```
that is adjancent to but independent from the `UnreadCountBadge` branches